<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>IPv4oETH</class>
 <widget class="QDialog" name="IPv4oETH">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>628</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>511</height>
    </rect>
   </property>
   <property name="title">
    <string>IPv4OnETH</string>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>130</y>
      <width>72</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>目的Mac：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ETH_SMAC">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>格式：00-11-22-33-44-55</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ETH_DMAC">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>120</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>源Mac：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>格式：00-11-22-33-44-55</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>格式：168.192.1.1</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>200</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>IPv4-源IP:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>400</y>
      <width>341</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>类型代码（例如：IPv4-0800）</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IPv4_DIP">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IPv4_L4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>360</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>IPv4-目的IP:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_IPv4_SIP">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>361</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>360</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>IPv4-4层报文:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>231</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>格式：168.192.1.1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ok">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>确定</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_cancel">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>460</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>取消</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
